src/data/ideas.js

```diff
@@ -0,0 +1,24 @@
+// ideas.js
+const axios = require('axios');
```

Comment:
Can you implement data fetching without `axios`, but with `fetch()` API?

See the documentation: https://nextjs.org/docs/app/building-your-application/data-fetching

Reply:
Yeah, I've read that. But thought I could write it without await and async. Fixed.